crm_phonecall_planner/README.rst

```diff
@@ -0,0 +1,120 @@
+#. Set the preferred phone calling schedule for the partner.
+#. Repeat above steps for all of your partners.
+
+Now, to actually generate the phone call planning:
+
+#. Go to *Sales > Phone Calls > Planner*.
+#. Fill the fields under *Call details*. Those fields will be saved literally
+   in the generated phone calls.
+#. Fill the fields under *Criteria*. Those fields are used to filter the
+   partners and the preexisting calls. The UTM fields will also be saved
+   literally in the generated phone calls.
+#. Fill the fields under *Times*. See note below.
+#. Fill the fields under *Repetition*. See note below.
+#. Press *Generate planning*.
+#. Wait a little bit (this is usually a long process).
+#. You will get to the list of planned phone calls. Start calling!
+
+Note about *Times* section
+--------------------------
+
+The *Start* and *End* times behave in a special way:
+
+* Their *date* part is used to know the start and end dates for the planning.
+* Their *time* part is used to know the time at which we will plan calls *each
+  day under the date range*.
+
+The *Call duration* field indicates the time spacing you want to leave between
+one call and the next one.
+
+So, for instance, if you select start on *2017-09-01 09:00:00*, end on
+*2017-09-03 10:00:00* and duration of *1:00*, it will try to generate these
+phone calls:
+
+* 2017-09-01 09:00:00
+* 2017-09-01 10:00:00
+* 2017-09-02 09:00:00
+* 2017-09-02 10:00:00
+* 2017-09-03 09:00:00
+* 2017-09-03 10:00:00
+
+Note about *Repetition* section
+-------------------------------
+
+If you choose not to repeat calls, the planner will try to schedule one single
+phone call for each **criteria combination** (*Partner + Campaign + Source +
+Medium*) under the specified *Times* conditions.
+
+If you choose instead to repeat calls after some amount of days, the planner
+will try to schedule before all the partners that you never called under
+current **criteria combination**, but if none is found it will repeat calls for
+such **criteria combination** if last call was done before the minimum
+*Days gap* you specified.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/111/9.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/crm/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Jairo Llopis <jairo.llopis@tecnativa.com>
+
+Do not contact contributors directly about support or help with technical issues.
+
+Funders
+-------
+
+The development of this module has been financially supported by:
```

Comment:
Remove this as it's not actually true (this is for final customers).

Reply:
Maybe we can propose a change in the template for grouping contributors by company, putting something like:
```rst
* Tecnativa (https://www.tecnativa.com)
  * Jairo Llopis <option e-mail>
...
```